components/infobox/wikis/arenafps/infobox_league_custom.lua

```diff
@@ -0,0 +1,169 @@
+---
+-- @Liquipedia
+-- wiki=arenafps
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Game = require('Module:Game')
+local Lua = require('Module:Lua')
+local PageLink = require('Module:Page')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local League = Lua.import('Module:Infobox/League', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+local Center = Widgets.Center
+
+local _args
+local _league
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local MODES = {
+	['arena'] = 'Arena',
+	['shaft arena'] = 'Shaft Arena',
+	['rocket arena'] = 'Rocket Arena',
+	['duel'] = 'Duel',
+	['sacrifice'] = 'Sacrifice',
+	['team deathmatch'] = 'Team Deathmatch',
+	['2vs2 tdm'] = '2vs2 Team Deathmatch',
+	['3vs3 circuit'] = '3vs3 Circuit',
+	['wipeout'] = 'Wipeout',
+	['race'] = 'Race',
+	['4vs4 team deathmatch'] = '4vs4 Team Deathmatch',
+	['ctf'] = 'Capture the Flag',
+	['free for all'] = 'Free For All',
+	['macguffin'] = 'MacGuffin',
+	['slipgate'] = 'Slipgate',
+	['clan arena'] = 'Clan Arena',
+}
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_league = league
+	_args = _league.args
+
+	_args.player_number = _args.participants_number
```

Comment:
should it only be alias or hard overwrite?

Reply:
current infobox only has participants_number